index.js

```diff
@@ -7,17 +7,18 @@ let hasBeenWarned = false;
 module.exports = {
   name: 'ember-router-service-polyfill',
 
-  included() {
+  included(includer) {
     this._super.included.apply(this, arguments);
 
     this._ensureThisImport();
 
     var checker = new VersionChecker(this);
     var emberVersion = checker.forEmber();
+    var includerIsApp = includer.name === this.project.name();
```

Comment:
I wonder if we can use `project.isEmberCLIAddon()`?

https://github.com/ember-cli/ember-cli/blob/master/lib/models/project.js